optionals/norealisticnames/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\ace\addons\norealisticnames
```

Comment:
```suggestion
z\ace\addons\norealisticnames

```

Reply:
We don't have newlines at EOF for any other component. Sure about this?

Reply:
We should have. :thinking:

Reply:
Some components have it, some don't, all can and should have it though. Mostly older ones don't have it as those files never get touched.